cli/plasmo/templates/static/react17/content-script-ui-mount.tsx

```diff
@@ -68,6 +68,14 @@ async function createShadowContainer() {
 
   const shadowHost = document.createElement("div")
 
+  if (typeof Mount.getShadowHostId() === "function") {
+    const SHADOW_HOST_ID =  await Mount.getShadowHostId();
+    // as per HTML5, element ids must not be empty & must not contain spaces
+    if (Mount.SHADOW_HOST_ID.length > 0 && Mount.SHADOW_HOST_ID.indexOf(' ') >= 0) {
```

Comment:
this should check the `SHADOW_HOST_ID`. Also, the naming conv for a variable should align with the rest of the codebase.